src/isofile-advanced-creation.js

```diff
@@ -192,7 +192,7 @@ ISOFile.prototype.addSample = function (track_id, data, _options) {
 	return sample;
 }
 
-ISOFile.createSingleSampleMoof = function(sample) {
+ISOFile.prototype.createSingleSampleMoof = function(sample) {
 	var moof = new BoxParser.moofBox();
 	moof.add("mfhd").set("sequence_number", this.nextMoofNumber);
 	this.nextMoofNumber++;
```

Comment:
code here uses `this.nextMoofNumber` and this context is lost because `createSingleSampleMoof` was defined as  a static method